literature.bib

```diff
@@ -832,6 +843,21 @@ @article{SBB+:SoSyM23
 	typo3Tags = {VariantSyncPub},
 }
 
+@inproceedings{RS:SPLC23,
+	author = {Rosiak, Kamil and Schaefer, Ina},
+	title = {{The e4CompareFramework: Annotation-based Software Product-Line Extraction}},
+	year = 2023,
+	isbn = {9798400700927},
+	publisher = ACM,
+	address = NY,
+	doi = {10.1145/3579028.3609012},
+	booktitle = SPLC,
+	pages = {34--38},
+	numpages = {5},
```

Comment:
remove braces

Reply:
I did a quick search for `numpages` and all entries have braces. I better not introduce an inconsistency now.

Reply:
@skrieter what do you think?

Reply:
I think the policy applied only to `year`.